test/ImportCommand.js

```diff
@@ -109,6 +109,22 @@ describe("ImportCommand", () => {
       expect(await pathExists(newFilePath)).toBe(true);
     });
 
+    it("skips empty patches with --flatten", async () => {
```

Comment:
Does it _require_ `--flatten` to skip the empty commits? The code (here or in #1059) doesn't seem to imply this.

Reply:
If I understand this correctly, the problem occurs only with `--flatten`, because `createPatchForCommit` uses `git log`, instead of `git format-patch`. These have different behavior for empty commits:

```
(~) git format-patch -1 f3121c --stdout                                             
(~) git log --reverse --first-parent -p -m --pretty=email --stat --binary -1 f3121c 
From f3121c... Mon Sep 17 00:00:00 2001
From: Alex Gyoshev <___@___.__>
Date: Thu, 12 Oct 2017 14:23:10 +0300
Subject: ...
(~) 
```

Reply:
Oh, interesting! We should probably address this disparity in the future.